src/main/java/BasketInformations.java

```diff
@@ -0,0 +1,98 @@
+package com.company;
+
+import java.util.HashMap;
+
+/**
+ * This class is a basket
+ */
+public class BasketInformations {
+
+	// The product of the basket
+	static HashMap<String, Integer> map = new HashMap<String, Integer>();
+
+	static String[] localProducts = new String[]{"123", "222", "44", "657"};
+
+	// The fact that the basket has promo code
+	private static boolean codeDePromotion = false;
```

Comment:
```suggestion
	private boolean hasPromoCode = false;
```

Since this field is static it is being shared across all instances of BasketInformations, I suggest converting it to an instance variable instead.

Also the variable name is not intuitive, I suggest changing it to something like `hasPromoCode`